internal/powerbi/resource_pbix.go

```diff
@@ -46,22 +46,26 @@ func ResourcePBIX() *schema.Resource {
 			},
 			"skip_report": {
 				Type:        schema.TypeBool,
-				Description: "If true only the PBIX dataset is deployed.",
+				Description: "If true, only the PBIX dataset is deployed.",
 				Optional:    true,
 				Default:     false,
 			},
 			"report_id": {
 				Type:        schema.TypeString,
 				Description: "The ID for the report that was deployed as part of the PBIX.",
-				Optional:    true,
 				Computed:    true,
 			},
 			"dataset_id": {
 				Type:        schema.TypeString,
 				Description: "The ID for the dataset that was deployed as part of the PBIX.",
-				Optional:    true,
```

Comment:
technically this is a breaking change, but I never really intended these to be settable